src/test/java/br/com/sbk/sbking/dto/LobbyScreenTableDTOTest.java

```diff
@@ -0,0 +1,53 @@
+package br.com.sbk.sbking.dto;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.Test;
+
+import br.com.sbk.sbking.core.Direction;
+import br.com.sbk.sbking.core.Player;
+import br.com.sbk.sbking.gui.screens.GameScreen;
+
+public class LobbyScreenTableDTOTest {
```

Comment:
Hmm, I wouldn't write tests for it, but as they are already written, it is good this way :)